features/kernel/src/main/feature/feature.xml

```diff
@@ -64,14 +64,15 @@
 
     <!-- Override the pax-web pax-http-whiteboard feature and remove the vulnerable pax-web-jsp bundle-->
     <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="${pax.web.version}_1">
-        <feature version="[6.0,6.1)">pax-http</feature>
+        <feature version="[7.2,7.3)">pax-http</feature>
         <bundle dependency="true" start-level="30">mvn:org.eclipse.jdt.core.compiler/ecj/4.5.1</bundle>
         <bundle start-level="30" dependency="true">mvn:javax.el/javax.el-api/3.0.0</bundle>
-        <!--<bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jsp/${pax.web.version}</bundle>-->
+        <!--<bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jsp/${project.version}</bundle>-->
```

Comment:
Is there a reason we are keeping this commented out bundle?

Reply:
This feature and its contents is copied from pax-web and then modified slightly. That's one of the modifications made, so I think we're just keeping it commented out to be explicit.